packages/sync/src/Defaults.php

```diff
@@ -623,6 +624,21 @@ public static function get_known_importers() {
 		return apply_filters( 'jetpack_sync_known_importers', self::$default_known_importers );
 	}
 
+	/**
+	 * Whether this is a system with a multiple networks.
+	 * We currently need this static wrapper because we statically define our default list of callables.
+	 *
+	 * @since 7.6.0
+	 *
+	 * @uses Automattic\Jetpack\Status::is_multi_network
+	 *
+	 * @return boolean
+	 */
+	public static function is_multi_network() {
```

Comment:
why not making the `Status` one static instead?

Reply:
No, I wouldn't want to do that, as I'm consciously running away from static methods when possible. The only reason I'm doing this is because of the static way we're declaring our default callable whitelist - I we can't initialize an object and use its method for a callable in that context. I'd very much like to change that definition so it happens in a method, but that's material for another PR.